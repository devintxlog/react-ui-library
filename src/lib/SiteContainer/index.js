import React from 'react'

import styles from './styles.module.scss'

//render prop component to be used for formatting other components
class SiteContainer extends React.Component {
  render(){
    return (
      <div className={styles.container}>
        {this.props.children}
      </div>
    )
  }
}

export default SiteContainer