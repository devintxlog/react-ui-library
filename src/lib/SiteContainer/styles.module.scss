.container {
  margin: 0 auto;
  width: 100%;
  max-width: 1000px;
  padding-left: 20px;
  padding-right: 20px;
}