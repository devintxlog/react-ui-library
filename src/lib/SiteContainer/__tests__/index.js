import React from 'react'

import { shallow } from 'enzyme'
import SiteContainer from '..'

describe('SiteContainer', () => {
  //define default props
  let props = {}

  //create a shallow wrapper than can be reused
  let wrapper

  //initialize the components
  beforeEach(() => {
    wrapper = shallow(<SiteContainer {...props}/>)
  })

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})