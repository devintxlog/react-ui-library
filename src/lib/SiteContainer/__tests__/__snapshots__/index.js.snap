// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteContainer renders correctly 1`] = `
<div
  className="container"
/>
`;
